unlock-app/src/__tests__/middlewares/storageMiddlewareHandler.test.js

```diff
@@ -0,0 +1,59 @@
+import * as storageMiddlewareHandler from '../../middlewares/storageMiddlewareHandler'
+
+const dispatch = jest.fn()
+
+xdescribe('createLock', () => {
+  describe('when the signature can be generated')
+  describe('when the signature cant be generated')
+})
+
+xdescribe('updateLock', () => {
+  jest.mock('../../services/storageService', () => {
+    let lockLookup = jest.fn(() => {
+      return Promise.resolve({ data: 'goldfish' })
+    })
+    return jest.fn().mockImplementation(() => {
+      return { lockLookup: lockLookup }
+    })
+  })
+
+  var existingAddress = '0x356454'
+  var nonexistingAddress = '0x02323'
+
+  xdescribe('when the lock can be found', () => {
+    it('fetches the lock details', done => {
+      const spy = jest.spyOn(dispatch, 'play')
+      storageMiddlewareHandler.updateLock(existingAddress, dispatch)
+      expect(spy).toBeCalled()
+      done()
+    })
+
+    it('dispatches an appropriate event', done => {
+      storageMiddlewareHandler.updateLock(existingAddress, dispatch)
+      expect(dispatch).toBeCalled()
+      done()
+    })
+  })
+
+  xdescribe('when the lock cant be found', () => {
+    it('dispatches an appropriate event', done => {
+      storageMiddlewareHandler.updateLock(nonexistingAddress, dispatch)
+      expect(dispatch).toBeCalled()
+      done()
+    })
+  })
+})
+
+xdescribe('lockDeployed', () => {
```

Comment:
Hum, it looks like `xdescribe` is like `describe.skip`. Can we use the later for consistency?

Reply:
Also is this really meant to be skipped?

Reply:
As part of the middleware adjusting, this object and associated tests no longer exist